src/stylesheets/components/_footer.scss

```diff
@@ -324,13 +365,17 @@
       }
 
       .usa-footer__primary-link {
-        &::before {
-          @include add-background-svg("usa-icons/navigate_next");
-        }
-
         @include at-media("mobile-lg") {
           margin: 0;
         }
+
+        @include place-icon($chevron-next, "before", 0.5, "auto", "default");
```

Comment:
Inlcude's should be first. Please move to `line 368` and have the media query come after.

Reply:
Resolved in [c1dae62](https://github.com/uswds/uswds/pull/4528/commits/c1dae62dac19c1f422211a5695d21d81ad914c57)